src/components/ProgramRecord/ProgramRecordHeader.jsx

```diff
@@ -77,15 +77,18 @@ function ProgramRecordHeader({
         </span>
       </div>
       <div className="learner-info my-3">
-        <FormattedMessage
-          id="learner.username"
-          defaultMessage="{username}"
-          description="Username of the learner"
-          values={{
-            username: learner.username,
-          }}
-        />
-        |
+        {
+          learner.full_name && (
+          <FormattedMessage
+            id="learner.full_name"
+            defaultMessage="{full_name}|"
```

Comment:
Maybe this is just me but I feel like we need non-breaking spaces around this pipe? Defer to UX, but it's hard to read the name in the screenshots, it just flows into the email address.

I know this was already a problem with username, but while we're here improving things...

Reply:
Yes, I agree.

Reply:
Done